packages/react-instantsearch/examples/next-app/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "create-next-example-app",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "build": "NODE_ENV=development next build",
```

Comment:
I am doing some chore work on the build and upgrades so I will update this one I think.